packages/firestore/test/integration/util/emulator_settings.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Projects used in emulator integration tests.
+export const EMULATOR_PROJECT_ID = 'test-emulator';
+export const ALT_PROJECT_ID = 'test-db2';
```

Comment:
Could this be merged with `settings.ts`?

Reply:
After I moved them to `settings.ts`, the failure reminded me why they are in a separate file: `settings.ts` itself imports relative path `../../../../../config/project.json`, which will not work in the rollup build.

I had to do this, and let the tree-shaking help to remove it.